### DISCUSSION TITLE
Isolate Changes to Specific Chunk/Hard-Name a chunk

### DISCUSSION THREAD

**Discussion by helloimalastair** [vercel/next.js#28795] [Help]
I have a project I am building, using [Ryosuke's](https://github.com/whoisryosuke) [Next MDX Deck Template](https://github.com/whoisryosuke/next-mdx-deck), where I have an interesting problem. I am attempting to build a WebSocket-based control solution using it, where anyone visiting the base URL will be shown the normal slideshow, but without any controls(keyboard or swipe), and anyone who visits a separate path(`/control`, for example) would be shown the slideshow, but with controls. They would both have a WebSocket connection, allowing anyone connected to `/control` to control the position of the slideshow for all connected clients. The only problem is, that the code for whether to respond to keyboard events or not is built into the deployed chunks by the build process. Is there any way to force that specific bit of code into a chunk with a hard-coded name, so that I can have one copy of the site, where I only serve the `KeyboardEvent` chunk if they are on the `/control` path, and otherwise omit it?

**Comment by adogg43** [comment]
do u jmndubwhoab

**Reply by helloimalastair** [reply]
What?
